src/javascript/JContent/EditFrame/Box.jsx

```diff
@@ -169,7 +195,12 @@ export const Box = React.memo(({
              className={clsx(styles.root)}
              style={currentOffset}
         >
-            <div className={clsx(styles.rel, isHeaderDisplayed ? boxStyle : styles.relNoHeader, isSelected ? styles.selected : styles.current)}>
+            <div className={clsx(styles.rel, isHeaderDisplayed ? boxStyle : styles.relNoHeader, isCurrent && !isSelected ? styles.current : '', isSelected ? styles.selected : '')}
+                 style={{
+                     '--colorCurrent': borderColorCurrent,
+                     '--colorSelected': borderColorSelected
+                 }}
+            >
                 {isHeaderDisplayed && Header}
 
                 {breadcrumbs.length > 0 &&
```

Comment:
I think it would be very useful to be able to customize that part (either defined a specific footer, or at least have an option to display or not display the breadcrumb)

Reply:
As we said, we will not do it for the moment, if we have a case where we need a custom footer we will add it at this moment